index.php

```diff
@@ -4,17 +4,18 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Party Hard</title>
+    <title>Party <?php echo "Hard"; ?></title>
```

Comment:
Dal bych tam finálně něco jiného.

Reply:
to si muze prava kazda pojmenovat uz po svem